docs/syntax-reference.md

```diff
@@ -337,6 +226,121 @@ The WITH clause supports the following properties:
 | TIMESTAMP               | Associates the message timestamp in the Kafka topic with a column in the KSQL table. Time-based operations such as windowing will process a record according to this timestamp. |
 
 
+### DESCRIBE
+
+**Synopsis**
+
+```sql
+DESCRIBE [EXTENDED] (stream_name|table_name);
```

Comment:
I collapsed DESCRIBE and DESCRIBE EXTENDED to make it easier to see it's "one" statement.